schemas/questionnaire_v1.json

```diff
@@ -55,8 +55,12 @@
       "description": "The survey (Census or Census Coverage Survey) that an address has been sampled into for participation"
     },
     "form_type": {
-      "enum": ["H", "I", "C"],
-      "description": "The form types are Household (H), Individual (I) or Communal Establishment (C)"
+      "enum": ["H", "I", "C", "business"],
```

Comment:
- `H`, `I` and `C` are census form types
 - a business form type can be any alphanumeric string

Reply:
Done, thanks!